test/e2e/test_cluster_configs/availabilityset.json

```diff
@@ -51,6 +53,16 @@
 					],
 					"availabilityProfile": "AvailabilitySet",
 					"vnetSubnetId": "/subscriptions/SUB_ID/resourceGroups/RG_NAME/providers/Microsoft.Network/virtualNetworks/VNET_NAME/subnets/SUBNET_NAME"
+				},
+				{
+					"name": "poolwin",
+					"count": 2,
+					"vmSize": "Standard_D2_v3",
+					"osType": "Windows",
```

Comment:
You need a windowsProfile section with at least admin undername and password

			"windowsProfile": {
				"adminUsername": "azureuser",
				"adminPassword": "replacepassword1234$",
				"enableAutomaticUpdates": false,
				"sshEnabled": true
			}
https://github.com/Azure/aks-engine/blob/0e54eceb35b6abde5c5f9ad2ba34826ba42c23e6/test/e2e/test_cluster_configs/everything.json#L128

Reply:
oh right, always bite me